### ISSUE TITLE
AttributeError: 'QWheelEvent' object has no attribute 'delta'

### ISSUE THREAD

**Issue by chenqi0805** [enthought/enable#361]
Qt5 compatibility bug:
```
Traceback (most recent call last):
  File "/Users/qchen/.edm/envs/lcd/lib/python3.6/site-packages/enable/qt4/base_window.py", line 232, in wheelEvent
    self.handler.wheelEvent(event)
  File "/Users/qchen/.edm/envs/lcd/lib/python3.6/site-packages/enable/qt4/base_window.py", line 122, in wheelEvent
    self._enable_window._handle_mouse_event("mouse_wheel", event)
  File "/Users/qchen/.edm/envs/lcd/lib/python3.6/site-packages/enable/abstract_window.py", line 301, in _handle_mouse_event
    mouse_event = self._create_mouse_event(event)
  File "/Users/qchen/.edm/envs/lcd/lib/python3.6/site-packages/enable/qt4/base_window.py", line 411, in _create_mouse_event
    delta = event.delta()
AttributeError: 'QWheelEvent' object has no attribute 'delta'
```

**Comment by mgrady3** [comment]
@chenqi0805 IIRC this is fixed in the current release of Enable: 4.8.0:
https://github.com/enthought/enable/blob/master/enable/qt4/base_window.py#L423

**Comment by corranwebster** [comment]
This should have been resolved by #302 and any version prior to that PR does not support Qt5.

I'm going to close the issue.  Please re-open if you still see the problem in 4.8.0 or in current master.
